infrastructure/staking-mechanics/rewards-payout.md

```diff
@@ -1,4 +1,236 @@
+- The variance in staking rewards increases as the total number of validators grows relative to the number of para-validators
+- In simpler terms, when more validators are added to the active set without increasing the para-validator pool, the disparity in rewards between validators becomes more pronounced
+
+However, despite this increased variance, rewards tend to even out over time due to the continuous rotation of para-validators across eras. The network's design ensures that over multiple eras, each validator has an equal opportunity to participate in para-validation, eventually leading to a balanced distribution of rewards.
+
+??? interface "Probability in Staking Rewards"
+
+    This should only serve as a high-level overview of the probabilistic nature for staking rewards.
+
+    Let:
+
+    - `pe` = para-validator era points
+    - `ne` = non-para-validator era points
+    - `EV` = expected value of staking rewards
+
+    Then, `EV(pe)` has more influence on the `EV` than `EV(ne)`.
+
+    Since `EV(pe)` has a more weighted probability on the `EV`, the increase in variance against the `EV` becomes apparent between the different validator pools (aka. validators in the active set and the ones chosen to para-validate).
+
+    Also, let:
+
+    - `v` = the variance of staking rewards
+    - `p` = number of para-validators
+    - `w` = number validators in the active set
+    - `e` = era
+
+    Then, `v` &#8593; if `w` &#8593;, as this reduces `p` : `w`, with respect to `e`.
+
+    Increased `v` is expected, and initially keeping `p` &#8595; using the same para-validator set for all parachains ensures [availability](TODO: add path) and [voting](https://wiki.polkadot.network/docs/learn-polkadot-opengov/){target=\_blank}. In addition, despite `v` &#8593; on an `e` to `e` basis, over time, the amount of rewards each validator receives will equal out based on the continuous selection of para-validators.
+
+    There are plans to scale the active para-validation set in the future.
+
+## Payout Scheme
+
+Validator rewards are distributed equally among all validators in the active set, regardless of the total stake behind each validator. However, individual payouts may differ based on the number of era points a validator has earned. Although factors like network connectivity can affect era points, well-performing validators should accumulate similar totals over time.
+
+Validators can also receive tips from users, which incentivize them to include certain transactions in their blocks. Validators retain 100% of these tips.
+
+Rewards are paid out in the network's native token (KSM for Kusama and DOT for Polkadot). 
+
+The following example illustrates a four member validator set with their names, amount they have staked, and how payout of rewards is divided. This scenario assumes all validators earned the same amount of era points and no one received tips: 
+
+<div class="mermaid">
```

Comment:
Just curious, why does this need to be in a div?

Reply:
As we have the mermaid plugin installed, we can directly use:

\```mermaid
\```


Since the plugin will convert that under the hood into:

\<div class="mermaid">
\</div>

I'd suggest that we always stick to using the first alternative, just for general consistency :)

Reply:
I think this was leftover from when I was trying to get the CSS right. I just checked the stylesheet though and I can't see anywhere I'm using the class to apply styling in the current setup. I will remove those and look for them when I do the site audit as well.